lms/static/styles/components/_URLPicker.scss

```diff
@@ -1,7 +0,0 @@
-.URLPicker__dialog {
-  display: flex;
-  flex-direction: column;
-  width: 80vw;
-  max-width: 500px;
```

Comment:
The `max-width: 500px` (and for `LMSFilePicker` too) were just a subjective judgement that it looked nicer to avoid having the dialog be too wide.

Reply:
Ah ok I can add that back in with a comment explaining that choice.

Reply:
Note in order to allow overriding of the width like this I had to add margin-left and right auto. I couldn't get it to center by using flex settings. I admit don't really understand why that was the case though.